docs/nuget-org/Deprecate-packages.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Deprecating packages on nuget.org
+description: Detailed description on the process of deprecating packages and how the clients shows this information
+author: anangaur
+ms.author: anangaur
+ms.date: 09/23/2019
+ms.topic: conceptual
+ms.reviewer: karann-msft
+---
+
+# Deprecating packages
+
+As a package publisher, if you no longer want to maintain a package or would like to move your package consumers to move to another package due to various reasons, deprecate your package. Package deprecation is different than `unlisting` your package as explained below:
```

Comment:
"or would like to **encourage** your package consumers to move to another package"